### PR TITLE
Add `osu!taiko Mapping World Cup 2024` news post and wiki page

### PR DESCRIPTION
**Pull request by Hiviexd**
[news post preview](https://osu3.roanh.dev/home/news/2024-05-19-osu-taiko-mapping-world-cup-2024-registrations), [#osu-news thread](https://discord.com/channels/188630481301012481/1238086969423298570)
[wiki article preview](https://osu3.roanh.dev/wiki/en/Contests/TMWC/2024)

- [x] contest listing

### CODE REVIEW

File: news/2024/2024-05-17-osu-taiko-mapping-world-cup-2024-registrations.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "osu!taiko Mapping World Cup 2024: Registrations Open!"
+date: 2024-05-17 18:00:00 +0000
+---
+
+Attention all osu!taiko mappers, gather up your fellow countrymen and prepare for a showdown never seen before!
```
Comment (0x84f) [inline news/2024/2024-05-17-osu-taiko-mapping-world-cup-2024-registrations.md:7]:
```suggestion
Attention all osu!taiko mappers, gather your fellow countrymen and prepare for a showdown never seen before!
```

File: news/2024/2024-05-17-osu-taiko-mapping-world-cup-2024-registrations.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+![](/wiki/Contests/TMWC/2024/img/banner.png)
+
+The osu!taiko Mapping World Cup is a country-based, multi-stage osu!taiko mapping contest on a scale unlike anything seen previously in the osu!taiko scene. Countries will battle it out over multiple rounds of mapping, completing specific challenges along the way, to decide who is definitively the best mapping country out there.
+
+Team sizes will range from 3 to 6 people, with the ability to have multiple teams per country depending on the amount of sign-ups.
+
+If you want to read more about the rules, contest format, schedule, and juding criteria, please check out this contest's [wiki page](/wiki/Contests/TMWC/2024) and [forum post](https://osu.ppy.sh/community/forums/topics/1865674) for more information.
```
Comment (0x84f) [inline news/2024/2024-05-17-osu-taiko-mapping-world-cup-2024-registrations.md:15]:
```suggestion
If you want to read more about the rules, contest format, schedule, and judging criteria, please check out this contest's [wiki page](/wiki/Contests/TMWC/2024) and [forum post](https://osu.ppy.sh/community/forums/topics/1865674) for more information.
```

File: news/2024/2024-05-17-osu-taiko-mapping-world-cup-2024-registrations.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+![TMWC 2024 badge](/wiki/Contests/TMWC/2024/img/badge.png "osu!taiko Mapping World Cup 2024 winning team badge")
+
+The amount of contest points awarded will be determined by the number of mappers participating in the contest, as detailed in the [contest points key](/wiki/Contests/Contest_points#points-key).
+
+---
+
+Please note that this is just a quick overview of the tournament. If you want to participate, please read the full rules, which can be found on the [wiki page](/wiki/Contests/TMWC/2024), or you can ask questions in our [Discord server](https://discord.gg/A33s9SU9Bt).
+
+Thanks for making it this far! We hope you are excited to share your passion for mapping through this contest and we encourage everyone to sign up. Best of luck!
```
Comment (0x84f) [inline news/2024/2024-05-17-osu-taiko-mapping-world-cup-2024-registrations.md:43]:
```suggestion
Thanks for making it this far! We hope you are excited to share your passion for mapping through this contest, and we encourage everyone to sign up. Best of luck!
```

File: wiki/Contests/TMWC/2024/en.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+| Event | Timestamp (UTC) |
+| --: | :-- |
+| Round 6 mapping phase | 2024-08-24 until 2024-09-07 |
+| Round 6 judging phase | 2024-09-07 until 2024-09-21 |
+| Results | 2024-09-21 |
+
+## Organisation
+
+The osu!taiko Mapping World Cup 2024 is running by various community members.
```
Comment (Walavouchey) [inline wiki/Contests/TMWC/2024/en.md:53]:
```suggestion
The osu!taiko Mapping World Cup 2024 is run by various community members.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/Contests/TMWC/2024/en.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+- **We expect everyone to conduct themselves with a sporting attitude and with integrity. If you share your creations outside of your team channel before judging is complete, or attempt any form of leaking confidential information, you will be disqualified.**
+
+### Judging criteria
+
+- **Flow/Playability (5):** Points will be assigned based on a clear correlation between hand movements and the song, as well as how the map plays.
+- **Judge's Impression (5):** Points will be assigned based on the judge's opinion of your submission.
+- **Ranking Criteria (5):** Points will be assigned based on how well the map follows the ranking criteria.
+- **Structure (10):** Points will be assigned based on consistency and how well the difficulty curve is constructed.
+- **Creativity (10):** Points will be assigned based on how the submission uses creative and fitting elements to make itself stand out.
+- **Relevance and modifier (15):** Points will be assigned based on how well the map fits the song and how well the round modifier is incorporated.
```
[the remark this answers is not in the document]
Comment (ikin5050) [reply wiki/Contests/TMWC/2024/en.md:110]: intention is title case so last one will have to be changed to Relevance and Modifier

File: wiki/Contests/TMWC/2024/en.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Cash prize, unique profile badge (pending), [contest points](/wiki/Contests/Contest_points) (pending) |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | Cash prize, contest points (pending) |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | Cash prize, contest points (pending) |
+
+![TMWC 2024 badge](img/badge.png "osu!taiko Mapping World Cup 2024 winning team badge")
```
[the remark this answers is not in the document]
Comment (Hiviexd) [reply wiki/Contests/TMWC/2024/en.md:69]: oo didn't know this template was a thing, i was following a semi-recent contest listing

will move it

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**